webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -94,6 +94,9 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
           this.contactsActions.removeContactFromList({ _id: change.id });
           this.hasContacts = this.contactsList.length;
         }
+        if (this.usersHomePlace && change.id === this.usersHomePlace._id) {
+          this.getUserHomePlaceSummary(change.id);
```

Comment:
```suggestion
         this.usersHomePlace = await this.getUserHomePlaceSummary(change.id);
```
This function is made to return the home place `getHomePlace....` instead of being a set `setHomePlace...`. Also it makes it closer to a pure function